charts/stable/resilio-sync/values.yaml

```diff
@@ -46,11 +45,11 @@ workload:
         main:
           probes:
             liveness:
-              type: http
-              path: /
+              type: tcp
+              port: "{{ .Values.service.main.ports.main.port }}"
```

Comment:
needs to be targetPort, or you need to remove targetPort from above. (both are fine)